client/src/locales/fr-FR.json

```diff
@@ -581,7 +582,14 @@
             "cancelled": "annulé",
             "failed": "échoué",
             "browse": "Aller à l'emplacement",
-            "failedDetails": "Vérifier votre connexion internet"
+            "failedDetails": "Vérifier votre connexion internet."
+        },
+        "RestoreFile": {
+            "restore": "Restauration de {name}",
```

Comment:
```suggestion
            "restore": "Restauré {name}",
```

Reply:
This is the message used while the file is being restored, or while in the queue to be restored.